File: Coding/coding modul 6/PRAK601-2410817110007-MUHAMMADHAMKAAKBARI.c
#include <stdio.h>

int main() {
    int baris, kolom;

    // Input jumlah baris dan kolom
    scanf("%d %d", &baris, &kolom);

    int matriks[baris][kolom]; // Deklarasi array 2D untuk menyimpan matriks

    // Input elemen matriks (baris * kolom angka)
    for (int i = 0; i < baris; i++) {
        for (int j = 0; j < kolom; j++) {
            scanf("%d", &matriks[i][j]);
        }
    }

    printf("\n");

    // Output matriks
    for (int i = 0; i < baris; i++) {
        for (int j = 0; j < kolom; j++) {
            printf("%d ", matriks[i][j]);
        }
        printf("\n"); // Pindah ke baris berikutnya
    }

    return 0;
}

File: Coding/coding modul 6/PRAK601-2410817110007-MUHAMMADHAMKAAKBARI.py
# Input jumlah baris dan kolom
baris, kolom = map(int, input("").split())

# Input elemen matriks
matriks = []  # List untuk menyimpan matriks
angka = list(map(int, input().split()))  # Input seluruh angka dalam satu baris

print("\n")

# Mengisi matriks
index = 0
for i in range(baris):
    row = []
    for j in range(kolom):
        row.append(angka[index])
        index += 1
    matriks.append(row)

# Output matriks
for row in matriks:
    print(" ".join(map(str, row)))
